### PR TITLE
Fix contract address for TEL

### PR DESCRIPTION
**Pull request by centromere**
See: https://telcoin.medium.com/smart-contract-update-faq-f6ff612c091f

### CODE REVIEW

**Comment by centromere** [comment]
Anyone?

**Comment by centromere** [comment]
Anyone?

**Comment by centromere** [comment]
Anyone?

File: tokens/eth/0x467bccd9d29f223bce8043b84e8c8b282827790f.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "symbol": "TEL",
-  "address": "0x85e076361cc813A908Ff672F9BAd1541474402b2",
```
Comment (Whiteme01) [inline tokens/eth/0x467bccd9d29f223bce8043b84e8c8b282827790f.json:3]: https://github.com/github/docs/blob/main/.github/actions-scripts/update-merge-queue-branch.js

Comment (Whiteme01) [reply tokens/eth/0x467bccd9d29f223bce8043b84e8c8b282827790f.json:3]: > https://github.com/github/docs/blob/main/.github/actions-scripts/update-merge-queue-branch.js

**Pull request closed without merging**